time/src/test/java/net/jqwik/time/api/DurationTests.java

```diff
@@ -496,7 +496,24 @@ void between6() {
 	@Group
 	class CheckEqualDistribution {
 
-		//TODO
+		@Property
+		@Disabled("Failing at the moment")
+		void seconds(@ForAll("durationsNear0") Duration duration) {
+			Statistics.label("Seconds")
+					  .collect(duration.getSeconds())
+					  .coverage(this::checkSecondsCoverage);
+		}
+
+		private void checkSecondsCoverage(StatisticsCoverage coverage) {
+			for (int value = -10; value <= 10; value++) {
+				coverage.check(value).percentage(p -> p >= 2.0);
+			}
+		}
```

Comment:
Do you have any idea why this test case is failing?

```
[CheckEqualDistribution:seconds] (1000) Seconds = 
    -10 (79) :  8 %
    10  (79) :  8 %
    0   (70) :  7 %
    -1  (53) :  5 %
    -2  (52) :  5 %
    4   (51) :  5 %
    -7  (49) :  5 %
    -4  (47) :  5 %
    -3  (45) :  5 %
    7   (45) :  5 %
    -8  (45) :  5 %
    8   (45) :  5 %
    2   (44) :  4 %
    1   (43) :  4 %
    -5  (42) :  4 %
    9   (42) :  4 %
    3   (35) :  4 %
    5   (34) :  3 %
    -9  (34) :  3 %
    -6  (33) :  3 %
    6   (33) :  3 %
```

```
CheckEqualDistribution:seconds = 
  org.opentest4j.AssertionFailedError:
    Percentage of 0.0 for [-10] does not fulfill condition for label "Seconds"
```

Reply:
Is checksSecondsCoverage() used in more than one place? Try if inlining the method will change the result.

Reply:
I've checked this already, it doesn't change the result:
```
Statistics.label("Seconds")
					  .collect(duration.getSeconds())
					  .coverage(c -> c.check(-5).percentage(p -> p >= 2.0));
```
`-5  (30) :  3 %`
`Percentage of 0.0 for [-5] does not fulfill condition for label "Seconds"`

Reply:
After some experimentation, I can say the following:
It has something to do with `duration.getSeconds()`

Fails:
```
Statistics.label("Seconds")
					  .collect(duration.getSeconds())
					  .coverage(c -> c.check(0).percentage(p -> p >= 2.0));
```
`0   (82) :  8 %`
`Percentage of 0.0 for [0] does not fulfill condition for label "Seconds"`

Works:
```
Statistics.label("Seconds")
					  .collect(duration.getNano())
					  .coverage(c -> c.check(0).percentage(p -> p >= 2.0));
```
`0         (57) :  5.70 %`

Reply:
I found the problem! int vs. long 

Feedback: Maybe jqwik should show that there is a comparison between int and long or show the datatype

Reply:
How should jqwik know for which value you are looking? You may collect longs and ints in the same statistics. I'll see if comparison can be more lenient.

Reply:
Okay that's true. But maybe `-10L (79) :  8 %` or `-10 (79 long) :  8 %` instead of `-10 (79) :  8 %`